crates/biome_js_analyze/src/lint/nursery/no_static_element_interactions.rs

```diff
@@ -0,0 +1,203 @@
+            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/keydown_event
+            "onKeyDown",
+            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/keyup_event
+            "onKeyUp",
+            //ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/keypress_event
+            "onKeyPress",
+        ],
+    ),
+    (
+        // ref: https://developer.mozilla.org/en-US/docs/Web/API/FocusEvent
+        "focus",
+        &[
+            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/focus_event
+            "onFocus",
+            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/blur_event
+            "onBlur",
+        ],
+    ),
+    (
+        // ref: https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent
+        "mouse",
+        &[
+            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event
+            "onClick",
+            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/contextmenu_event
+            "onContextMenu",
+            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/dblclick_event
+            "onDblClick",
+            "onDoubleClick",
+            // ref: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/drag_event
+            "onDrag",
+            // ref: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dragend_event
+            "onDragEnd",
+            // ref: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dragenter_event
+            "onDragEnter",
+            // ref: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dragleave_event
+            "onDragLeave",
+            "onDragExit",
+            // ref: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dragover_event
+            "onDragOver",
+            // ref: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dragstart_event
+            "onDragStart",
+            // ref: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/drop_event
+            "onDrop",
+            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/mousedown_event
+            "onMouseDown",
+            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/mouseenter_event
+            "onMouseEnter",
+            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/mouseleave_event
+            "onMouseLeave",
+            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/mousemove_event
+            "onMouseMove",
+            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/mouseout_event
+            "onMouseOut",
+            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/mouseover_event
+            "onMouseOver",
+            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/mouseup_event
+            "onMouseUp",
+        ],
+    ),
+];
```

Comment:
i think we don't need the `ref` comments, because they are obvious and redundant.
```suggestion
const EVENT_TO_HANDLERS: &[(&str, &[&str])] = &[
    (
        // ref https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent
        "keyboard",
        &[
            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/keydown_event
            "onKeyDown",
            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/keyup_event
            "onKeyUp",
            //ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/keypress_event
            "onKeyPress",
        ],
    ),
    (
        // ref: https://developer.mozilla.org/en-US/docs/Web/API/FocusEvent
        "focus",
        &[
            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/focus_event
            "onFocus",
            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/blur_event
            "onBlur",
        ],
    ),
    (
        // ref: https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent
        "mouse",
        &[
            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event
            "onClick",
            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/contextmenu_event
            "onContextMenu",
            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/dblclick_event
            "onDblClick",
            "onDoubleClick",
            // ref: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/drag_event
            "onDrag",
            // ref: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dragend_event
            "onDragEnd",
            // ref: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dragenter_event
            "onDragEnter",
            // ref: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dragleave_event
            "onDragLeave",
            "onDragExit",
            // ref: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dragover_event
            "onDragOver",
            // ref: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dragstart_event
            "onDragStart",
            // ref: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/drop_event
            "onDrop",
            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/mousedown_event
            "onMouseDown",
            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/mouseenter_event
            "onMouseEnter",
            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/mouseleave_event
            "onMouseLeave",
            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/mousemove_event
            "onMouseMove",
            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/mouseout_event
            "onMouseOut",
            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/mouseover_event
            "onMouseOver",
            // ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/mouseup_event
            "onMouseUp",
        ],
    ),
];
```

Reply:
@unvalley 
thank you,
i fixed in this commit!
[d09548f](https://github.com/biomejs/biome/pull/2981/commits/d09548f79268bbcb6bfc8e751f63e2409899acc9)